src/tests/condense-number.test.ts

```diff
@@ -52,4 +52,12 @@ describe('condenseNumber()', () => {
   it('does not apply precision to Intl formatting when the locale is not supported', () => {
     expect(condenseNumber(-150000, 'IN', {maxPrecision: 2})).toBe('-150.000');
   });
+
+  it('condenses numbers when given a locale and a region', () => {
```

Comment:
Probably better to test something like 'it falls back to the locale if a region is specified but not specifically handled'? And then you could pass something like `th-ZZZ` to assert that it will fall back to the "default" Thai formatting